components/LoginLogout.tsx

```diff
@@ -5,7 +5,7 @@ import { useEffect, useState, useContext } from "react";
 import type { Session } from 'next-auth';
 import Link from 'next/link';
 import RudderContext from './RudderContext';
-import { getAndSetAnonymousIdFromLocalStorage } from '../utils/url_utils';
+import { getAndSetAnonymousIdFromLocalStorage } from '../utils/rudderstack_initialize';
```

Comment:
I am not sure why GitHub is showing these changes here. I have double checked that both the branches, `akg/demo_banner` and `akg/banner_events` have these changes from master. In fact, I have branched out `akg/banner_events` after I had pulled main into `akg/demo_banner`. 

Please ignore these.